keycloak/keycloak-admin-ui
# Clients(authorization -> evaluate): Adds expandable results table

## Motivation

https://marvelapp.com/prototype/5eg9ic7/screen/83188837

## Brief Description

Adds expandable table for results of resource evaluation. 

## Verification Steps



1. Go to Clients
2. Create a new OIDC client and make sure "Authorization" and "Authentication" switches are both enabled.
3. Go to Authorization -> Evaluate
4. Fill out the fields and Click "Evaluate"
5. Verify the data is displayed correctly.
